site/components/pages/Infinity/Cards/cards.ts

```diff
@@ -10,60 +18,103 @@ export const cards = [
     description:
       "We're proud to offset 10+ years of carbon emissions from Olympus to become Climate Positive. From the Ohmies, with Love.",
     tonsRetired: 33333,
-    date: "14 MAY 2022",
+    date: "21 APR 2022",
   },
   {
     logo: polygonLogo,
     link: "https://www.klimadao.finance/retirements/0x087a7afb6975a2837453be685eb6272576c0bc06",
     description:
-      "We care deeply about our work, and in turn we care deeply about preserving it. This also applies more generally, and we wish to preserve the planet which has afforded us this freedom.",
-    tonsRetired: 65201,
-    date: "10 MAY 2022",
+      "Together with Polygon, the broader blockchain industry needs to form a united front to fund, support, and leverage technoilogy that helps heal the earth, rather than destroy it",
+    tonsRetired: 104794,
+    date: "21 JUN 2022",
   },
   {
     logo: markCubanLogo,
     link: "https://www.klimadao.finance/retirements/markcuban.klima",
     description: "Retire to offset corporate carbon usage.",
-    tonsRetired: 1100,
-    date: "4 APRIL 2022",
+    tonsRetired: 1000,
+    date: "04 APR 2022",
   },
   {
     logo: stadiusLogo,
     link: "https://www.klimadao.finance/retirements/stadius.eth",
     description:
-      "Offsetting team and operational carbon emissions since founding.",
+      "'Unless someone like you cares a whole awful lot Nothing is going to get better. It's not.' - The Lorax",
     tonsRetired: 75,
-    date: "6 APRIL 2022",
+    date: "06 APR 2022",
   },
   {
-    logo: olympusLogo,
-    link: "https://www.klimadao.finance/retirements/0xe06efa3d9ee6923240ee1195a16ddd96b5cce8f7",
+    logo: offsetraLogo,
+    link: "https://www.klimadao.finance/retirements/0xa83148b098038f366534253704ae4ff3cf931b94",
     description:
-      "We're proud to offset 10+ years of carbon emissions from Olympus to become Climate Positive. From the Ohmies, with Love.",
-    tonsRetired: 33333,
-    date: "14 MAY 2022",
+      "Offsetra is proud to pledge to offset 484 tonnes from our operational emissions, including employee travel. KlimaDAO has initiated a paradigm shift in the voluntary carbon market and we are excited to contribute to the on-chain market's innovations in the years to come.",
+    tonsRetired: 510,
+    date: "26 MAR 2022",
+    width: 120,
+    height: 30,
```

Comment:
I would rename this to: `logoWidth` and `logoHeight`

otherwise this reads as a value for the whole card

Reply:
updated